src/plone/jsonserializer/deserializer/converters.py

```diff
@@ -74,12 +74,17 @@ def default_converter(value, field):
 @adapter(Interface, IBool)
 @implementer(ISchemaCompatible)
 def bool_converter(value, field):
+    # since zope.interface >= 5.0 we never get here ... see below from_unicode_converter
     return bool(value)
 
 
 @adapter(Interface, IFromUnicode)
 @implementer(ISchemaCompatible)
 def from_unicode_converter(value, field):
+    # zope.schema.Bool fields also provide IFromUnicode
+    # since zope.interface >= 5.0 we get this adapter for boolean fields
+    if isinstance(value, bool):
+        value = str(value)
```

Comment:
Here we want a boolean and we already have one. So shouldn't this simply `return value`? Now it looks like for example:

1. value = True
2. `value = str(value)` -> 'True'
3. field.fromUnicode(value) -> return True